README.adoc

```diff
@@ -96,7 +96,8 @@ Versioned content is stored in branches that track the version of Redpanda Platf
 
 The production {url-playbook}[docs site playbook] instructs Antora to automatically aggregate content in the following branches:
 
-- `main`: Content for the latest version of Redpanda.
+- `main`: Content for the latest version of Redpanda Enterprise.
+- `redpanda-cloud`: Content for the latest version of Redpanda Cloud.
```

Comment:
```suggestion
```

Redpanda Cloud is now a repo 😄

Reply:
@JakeSCahill Can you please help me with the wording here for this Readme & for the redpanda-cloud Readme? So this is the list of branches off main. Should we have a separate list of repos off main to include labs and redpanda-cloud? And when we talk about the main branch, should we just use Redpanda (instead of Redpanda Platform)? 

- `main`: Content for the latest version of Redpanda Platform.
- `v/*`: Content for previous versions of Redpanda.
- `shared`: Content that is shared across all versions (asciidoc attributes and terms)
- `api`: API docs as well as the source OpenAPI spec files.
- `site-search`: The global site search page.